README.md

```diff
@@ -1,2 +1,183 @@
+
+*  **RHEL based image**
+
+    This image is not available as trusted build in [Docker Index](https://index.docker.io).
+
+    To test a rhel7-based php-5.5 image, you need to run the test on a properly
+    subscribed RHEL machine.
+
+    ```
+    $ cd sti-php
+    $ make test TARGET=rhel7 VERSION=5.5
+    ```
+
+*  **CentOS based image**
+
+    ```
+    $ cd sti-php
+    $ make test VERSION=5.5
+    ```
+
+**Notice: By omitting the `VERSION` parameter, the build/test action will be performed
+on all the supported versions of PHP. Since we currently only support version `5.5`
+you can omit this parameter.**
+
+
+Repository organization
+------------------------
+* **`<php-version>`**
+
+    * **Dockerfile**
+
+        CentOS based Dockerfile.
+
+    * **Dockerfile.rhel7**
+
+        RHEL based Dockerfile. In order to perform build or test actions on this
+        Dockerfile you need to run the action on properly subscribed RHEL machine.
+
+    * **`.sti/bin/`**
+
+        This folder contains scripts that are run by [STI](https://github.com/openshift/source-to-image):
+
+        *   **assemble**
+
+            Is used to install the sources into location from where the application
+            will be run and prepare the application for deployment (eg. installing
+            modules using npm, etc..)
+
+        *   **run**
+
+            This script is responsible for running the application, by using the
+            application web server.
+
+    * **`contrib/`**
+
+        This folder contains file with commonly used modules.
+
+    * **`test/`**
+
+        This folder contains a simple [STI](https://github.com/openshift/source-to-image)
+        test framework with a sample PHP app.
+
+        * **`test-app/`**
+
+            Simple PHP app for used for testing purposes in the [STI](https://github.com/openshift/source-to-image) test framework.
+
+        * **run**
+
+            Script that runs the [STI](https://github.com/openshift/source-to-image) test framework.
+
+* **`hack/`**
+
+    Folder contains scripts which are responsible for build and test actions performed by the `Makefile`.
+
+Image name structure
+------------------------
+##### Structure: openshift/1-2-3
+
+1. Platform name - php
+2. Platform version(without dots)
+3. Base builder image - centos7/rhel7
+
+Examples: `openshift/php-55-centos7`, `openshift/php-55-rhel7`
+
+Environment variables
+---------------------
+
+To set these environment variables, you can place them into `.sti/environment`
+file inside your source code repository.
+
+The following environment variables set their equivalent property value in the php.ini file:
+* **ERROR_REPORTING**: Default=E_ALL & ~E_NOTICE
+* **DISPLAY_ERRORS**: Default=ON
+* **DISPLAY_STARTUP_ERRORS**: Default=OFF
+* **TRACK_ERRORS**: Default=OFF
+* **HTML_ERRORS**: Default=ON
+* **INCLUDE_PATH**: Default=.:/opt/openshift/src:/opt/rh/php55/root/usr/share/pear
+* **SESSION_PATH**: Default=/tmp/sessions
```

Comment:
Be more descriptive regarding those flags, at least one sentence describing purpose of those vars.